src/csharp/Microsoft.Spark/Sql/Column.cs

```diff
@@ -592,6 +592,16 @@ public Column As(IEnumerable<string> alias)
             return Apply("as", alias);
         }
 
+        /// <summary>
+        /// Returns a column with values modified as per input object.
+        /// </summary>
+        /// <param name="colObject">Column object to apply</param>
+        /// <returns>Column object</returns>
+        public Column Apply(object colObject)
```

Comment:
If we're revealing this public method `Apply()` then I think we should rename the private method `Apply()` which has totally different semantics.

Reply:
Any suggestions on what to call the private Apply function?

Reply:
How about `ApplyMethod`